lib/tools/adb-emu-commands.js

```diff
@@ -31,6 +34,25 @@ emuMethods.rotate = async function () {
   await this.adbExecEmu(['rotate']);
 };
 
+emuMethods.powerAC = async function (state = 'on') {
+  if (POWER_AC_STATES.indexOf(state) === -1) {
+    log.errorAndThrow(`Wrong power AC state sent ${state}`);
+  }
+  await this.adbExecEmu(['power', 'ac', state]);
+};
+
+emuMethods.powerCapacity = async function (percent = 100) {
+  if (!parseInt(percent) || percent < 0 || percent > 100) {
```

Comment:
`parseInt(0, 10)` will be `false`. Do we want to be able to set it to `0`? 

I think you want something like `if (_.isNaN(percent) || percent < 0 || percent > 100) {` (assuming that `percent` has already been parsed).